tests/modules/aioseop/test-All_in_One_SEO_Pack-aiospMrtGetUrl.php

```diff
@@ -72,26 +76,24 @@ public function setUp() {
 		 * @since 2.4.4.1
 		 *
 		 * @param int $how_many
-		 * @return int[]
 		 */
-		public function setup_posts( $how_many = 0 ) {
+		public function setup_posts_aiospMrtGetUrl( $how_many = 0 ) {
```

Comment:
@EkoJr could you possibly reuse the method already defined in the base class or maybe modify it so that you don't have to defined your own setup for posts?

Reply:
Unfortunately, the post ID was needed for the tests, and all the base methods were set as final, so there's no way to override, and modifying the return value (from permalinks to IDs) seems like it would have unintended implications with other tests.

Still unsure about this one.